### PR TITLE
Add libv8, which is needed by r-v8 and some other CRAN packages

### PR DESCRIPTION
**Pull request by dpryan79**
This particular version is needed by CRAN packages, chromium uses much newer versions.

### CODE REVIEW

**Comment by dpryan79** [comment]
I'm curious if this builds on OSX. If so, I'll fix it to a single commit zip file with a checksum so it can be built more reproducibly.

**Comment by bgruening** [comment]
Great. xref: https://github.com/conda-forge/staged-recipes/issues/4523

**Comment by dpryan79** [comment]
Interesting, I hadn't seen that.

**Comment by dpryan79** [comment]
Hmm, it built on Linux locally (though not exactly in a clean environment)...

File: recipes/libv8/build.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
+cd build
+git clone https://chromium.googlesource.com/external/gyp
+cd ..
+sed -i.bak "s/-Werror//g" build/standalone.gypi
+sed -i.bak "s/-Werror//g" build/common.gypi
+python2 build/gyp_v8
+export CPP_INCLUDE_PATH=../src/third_party
+export CPLUS_INCLUDE_PATH=../src/third_party
```
Comment (bgruening) [inline recipes/libv8/build.sh]: export CPLUS_INCLUDE_PATH="${PREFIX}/include:${CPLUS_INCLUDE_PATH}"?

Comment (dpryan79) [reply recipes/libv8/build.sh]: It's not set to begin with.

**Pull request closed without merging**